README.md

```diff
@@ -1,18 +1,19 @@
 [![General Assembly Logo](https://camo.githubusercontent.com/1a91b05b8f4d44b5bbfb83abac2b0996d8e26c92/687474703a2f2f692e696d6775722e636f6d2f6b6538555354712e706e67)](https://generalassemb.ly/education/web-development-immersive)
 
-# jQuery AJAX GET Element
+# jQuery AJAX Delete Element
 
 ## Prerequisites
 
 -   [http-study](https://github.com/ga-wdi-boston/http-study)
 -   [json-study](https://github.com/ga-wdi-boston/json-study)
 -   [Ajax Get Collection](https://github.com/ga-wdi-boston/jquery-ajax-get-collection)
+-   [Ajax Get Element](https://github.com/ga-wdi-boston/jquery-ajax-get-element)
```

Comment:
In general we only include the last prereq. So Get Element would be the only prerequisite here, and the others are prereqs of Get Collection.